src/Widgets/DataFormUploadWidget/DataFormUploadWidgetComponent.tsx

```diff
@@ -11,6 +11,7 @@ import { useDropzone } from 'react-dropzone'
 import { Attachment } from '../../Components/Attachment'
 import { useCallback, useEffect, useState } from 'react'
 import prettyBytes from 'pretty-bytes'
+import { pseudoRandom32CharHex } from '../../utils/pseudoRandom32CharHex'
```

Comment:
* [ ] FYI another method could be an incrementing counter, which is mentioned alongside the random approach in https://react.dev/learn/rendering-lists#where-to-get-your-key.

Reply:
[crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) looks nice, but is only supported in newer Safari version (15.4+). I'm leaving it for now.